### PR TITLE
Add MEL dump function

### PR DESCRIPTION
**Pull request by Wenxing-hou**

### CODE REVIEW

File: spdm_dump/spdm/spdm_dump_spdm.c [hunk 1]
```diff
@@ -1847,6 +1851,101 @@ void dump_spdm_measurements(const void *buffer, size_t buffer_size)
...
+    message_size = sizeof(spdm_get_measurement_extension_log_request_t);
+    if (buffer_size < message_size) {
+        printf("\n");
+        return;
+    }
+
+    spdm_request = buffer;
+
+    if (!m_param_quite_mode) {
+        printf(", Offset=0x%x, Length=0x%x) ",
```
Comment (jyao1) [inline spdm_dump/spdm/spdm_dump_spdm.c:1870]: use `0x%08x`

File: spdm_dump/spdm/spdm_dump_spdm.c [hunk 2]
```diff
@@ -1847,6 +1851,101 @@ void dump_spdm_measurements(const void *buffer, size_t buffer_size)
...
+
+    spdm_response = buffer;
+    message_size += spdm_response->portion_length;
+    if (buffer_size < message_size) {
+        printf("\n");
+        return;
+    }
+
+    if (!m_param_quite_mode) {
+        printf(", PortLen=0x%x, RemLen=0x%x) ",
```
Comment (jyao1) [inline spdm_dump/spdm/spdm_dump_spdm.c:1904]: use `0x%08x`

File: spdm_dump/spdm/spdm_dump_spdm.c [hunk 3]
```diff
@@ -1847,6 +1851,101 @@ void dump_spdm_measurements(const void *buffer, size_t buffer_size)
...
+            if (m_spdm_mel_buffer_size >= spdm_mel->mel_entries_len + sizeof(spdm_measurement_extension_log_dmtf_t)) {
+                printf("\n    SpdmMelNumber(0x%08x)",
+                       spdm_mel->number_of_entries);
+                printf("\n    SpdmMelTotalLen(0x%08x)",
+                       spdm_mel->mel_entries_len);
+
+                mel_entry = (spdm_mel_entry_dmtf_t *)(spdm_mel + 1);
+                for (mel_index = 0; mel_index < spdm_mel->number_of_entries; mel_index++) {
+                    printf("\n    The %d MEL entry is:", mel_index);
+                    printf("\n        The MEL entry MELIndex is: %d", mel_entry->mel_index);
```
Comment (jyao1) [inline spdm_dump/spdm/spdm_dump_spdm.c:1933]: use `0x%08x`

File: spdm_dump/spdm/spdm_dump_spdm.c [hunk 4]
```diff
@@ -1847,6 +1851,101 @@ void dump_spdm_measurements(const void *buffer, size_t buffer_size)
...
+                    printf("\n        The MEL entry MeasIndex is: %d", mel_entry->meas_index);
```
Comment (jyao1) [inline spdm_dump/spdm/spdm_dump_spdm.c:1934]: use `0x%08x`

File: spdm_dump/spdm/spdm_dump_spdm.c (hunk 1 above)
Comment (Wenxing-hou) [reply spdm_dump/spdm/spdm_dump_spdm.c]: Thanks. I have fixed it.

File: spdm_dump/spdm/spdm_dump_spdm.c (hunk 3 above)
Comment (Wenxing-hou) [reply spdm_dump/spdm/spdm_dump_spdm.c]: Thanks. I have fixed it.

**Pull request merged**